manifests/c/Cortex/Cortex/0.3.2-beta/Cortex.Cortex.installer.yaml

```diff
@@ -6,7 +6,7 @@ PackageVersion: 0.3.2-beta
 Installers:
 - Architecture: x64
   InstallerType: nullsoft
-  InstallerUrl: https://download.crtx.gg/Cortex%20Setup%20latest.exe
+  InstallerUrl: https://storage.googleapis.com/crtx-prod.appspot.com/public/releases/Cortex%20Setup%20latest.exe
```

Comment:
Why this change? At https://docs.crtx.gg/getting-started/installation the download URL still shows as https://download.crtx.gg/Cortex%20Setup%20latest.exe

Reply:
URL is a redirect to `https://storage.googleapis.com/crtx-prod.appspot.com/public/releases/Cortex%20Setup%20latest.exe`.

I'm fine with closing the PR if it's not a needed change.

Reply:
I'd think it'd be better to keep the original URL since we support redirects in the client. If the publisher decides they want to redirect their URL to some other location at some point, this gives them that flexibility.